client/modules/IDE/components/CollectionList/CollectionListRow.jsx

```diff
@@ -198,9 +175,10 @@ class CollectionListRowBase extends React.Component {
     );
   };
 
-  renderCollectionName = () => {
-    const { collection, username } = this.props;
-    const { renameOpen, renameValue } = this.state;
+  const renderCollectionName = () => {
+    const { collection, username } = props;
+    // eslint-disable-next-line no-shadow, no-use-before-define
+    const { renameOpen } = renameOpen;
```

Comment:
This is definitely not right!  Sometimes it's okay to ignore eslint errors, but often they are there for a reason, to indicate that there's a problem with your code.

Reply:
i will remove that comment and commit again please reviewing my PR again

Reply:
please check now.. i have fixed all the eslint lines and ran the lint and tests they are no errors please check my PR and merge it.. I am waiting for it for so long

Reply:
> I am waiting for it for so long
Wtf it’s been like a week.  I have PRs of my own which are still awaiting review after 6 months.  The more you nag me about this the less it makes me want to work on your PRs.  I don’t want to volunteer my time for someone who is ungrateful and demanding.

Reply:
i am just asking to review my updated changes.. if it is bothering you. sorry and have a great day ahead. Don't disturb you again. take your time!